release/go-mod-messaging.yaml

```diff
@@ -1,10 +1,10 @@
 ---
 name: 'go-mod-messaging'
-version: '2.1.0'
+version: '2.1.1'
 releaseName: 'jakarta'
-releaseStream: 'main'
+releaseStream: 'jakarta'
 lts: true
```

Comment:
Curious if this need to still be true since all the branching, vendoring, etc. for the LTS is already in place.